### ISSUE TITLE
systemctl can't start indiwebmanager.service

### ISSUE THREAD

**Issue by guilleortas** [knro/indiwebmanager#63]
Hi, I couldn't find any issues regarding this. The problem I'm facing is that systemctl cannot start the service.

indiweb is installed and works just fine when called normally from the terminal. However, the command 'sudo systemctl status indiwebmanager.service' shows 'inactive (dead)' I can't get it work start normally. I've tried restarting the service, disabling it, rebooting the device a bunch of times and completely removing the service from systemctl, but nothing seems to work.

I want to clarify this is the second install I performed of indiweb. The first worked without issue. Both were done on RPi 4b 4GB.

Any ideas? Thanks!

**Comment by bluthen** [comment]
Maybe can see why if you do:
`sudo journalctl -u indiwebmanager`

**Comment by guilleortas** [comment]
There is nothing at all on the journald log

**Comment by guilleortas** [comment]
yep, I had set my username on there

**Comment by bluthen** [comment]
What happens if you try to run it manually then?

As your user:
```
/usr/local/bin/indi-web -v 
```

**Comment by guilleortas** [comment]
If I launch it directly using the full path as my user it works without issue.
I have created a version of the systemctl service changing the executable to just "echo helloworld" and does not work either, any ideas?
The status of the service always stays at 'inactive (dead)' and there are no logs

**Comment by bluthen** [comment]
start copy and pasting stuff, status, your actual file, etc.

**Comment by bluthen** [comment]
Maybe there is an issue with it starting enabled service at startup? What if you start it manually using systemctl? Do that start it?

`sudo systemctl start indiwebmanager`

**Comment by guilleortas** [comment]
Yes, I'm using Raspberry Pi OS, not using any virtual environments as per the readme instructions.
When I run ```sudo systemctl start indiwebmanager``` it never returns, seems like it never runs/starts the service

**Comment by bluthen** [comment]
I'm not sure why it behaving so differently for you. Maybe I'll have a fire up the latest raspberry pi os and try it out.

What if change type to simple?  idle I believe wait until everything has started, maybe you got something else not starting?

```
Type=simple
```

**Comment by guilleortas** [comment]
Yeah, no dice. Thank you so much for your help, though.
At this point I'm just starting it manually every time the system boots up.

**Comment by jfwells** [comment]
I had the same problem on rpi-os, it was systemd trying to start wayvnc, which never completed as I'm using X11 (and realVNC). 

You can see what is causing your systemd to hang using `systemd-analyze`. If it's wayvnc, then disable that service then restart.
